### ISSUE TITLE
Internal SourceMap causing CSS file size bloat.

### ISSUE THREAD

**Issue by BerzerkNinja** [madskristensen/WebCompiler#118]
Hi all,

This might be a carry on of #95

tldr; Can we change the source map to be an external file again? The source maps generated is too big, and is bloating out the CSS file size.

I'm having trouble with Source Maps for SCSS files. If I enable source mapping "sourceMap": true, it seems to add an inline source mapping. I.E.
/*# sourceMappingURL=data:application/json;base64,ewoJInZlcnNpb24.....

The SCSS file I'm using is a combo of most of the SCSS we use. This generates a large CSS file (~400kb). 

When creating a source map manually the external .map file is ~200kb. When creating a source map using WebCompiler the internal base 64 source map is ~ 500kb.

Is there an option to move the source map out into its own .map file instead I.E. using /*# sourceMappingURL=style.css.map */ instead of /*# sourceMappingURL=data:application/json;base64,ewoJInZlcnNpb24....."

**Comment by BerzerkNinja** [comment]
Was a dupe of #105
